APCAppCore/APCAppCore/Library/Scheduler/APCScheduler.m

```diff
@@ -1123,29 +368,33 @@ - (void) fetchTasksAndSchedulesFromServerAndThenUseThisQueue: (NSOperationQueue
         else
         {
             /*
-             Bounce over to the Sage SDK's thread, call the server, and then come
+             Bounce over to the Bridge SDK's thread, call the server, and then come
              back to our thread a while later.
              */
-            [SBBComponent (SBBScheduleManager) getSchedulesWithCompletion: ^(SBBResourceList *schedulesList,
-                                                                             NSError *errorFetchingSchedules)
+            NSDate *futureDate = [[NSDate date] dateByAddingDays:3]; // TODO: store this variable somewhere, can it be in the api?
```

Comment:
this will go away with the API update to daysAhead=xxx&offset=yyy; you'll just pass 3 to the new SDK method for the daysAhead parameter and it will figure out and send the correct time zone offset. Pending Alx finishing the backend support and getting it deployed to staging.